app/views/layouts/application.html.erb

```diff
@@ -9,7 +13,7 @@
   </head>
 
   <body>
-    <%= render 'layouts/site-nav' %>
+    <%= render 'layouts/header' %>
```

Comment:
The header partial is created in the next commit so adding the header in this commit would effectively break functionality. I would add this line in your next commit.